automated/linux/armnn-benchmarks/armnn-mlperf.sh

```diff
@@ -1,43 +1,65 @@
 #!/bin/sh
 
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+RESULT_FILE="${OUTPUT}/result.txt"
+
+usage() {
+    echo "Usage: $0 [-s <true|false>]" 1>&2
+    exit 1
+}
+
+while getopts "s:" o; do
+  case "$o" in
+    s) SKIP_INSTALL="${OPTARG}" ;;
+    *) usage ;;
+  esac
+done
+
+! check_root && error_msg "You need to be root to run this script."
+create_out_dir "${OUTPUT}"
+
+pkgs="ntp git wget curl libz-dev gcc g++ autoconf autogen libtool python3 python3-pip libjpeg-dev libtinfo5 libncurses5-dev libncursesw5-dev libboost-all-dev"
 dhclient
-apt-get install -y ntp
-apt-get --allow-releaseinfo-change update
-wget https://snapshots.linaro.org/componenets/armnn/latest-master/armnn.tar.xz
+install_deps "${pkgs}"
+
+export SNAPSHOT='https://snapshots.linaro.org/componenets/armnn/latest-master/armnn.tar.xz'
```

Comment:
this is still hardcoded. It should be passed to the script as parameter. Also please don't use 'latest', it can bring very surprising errors.